config.sample.js

```diff
@@ -3,5 +3,7 @@ module.exports = {
     appId: '',
     publicKey: '',
     apiKey: '',
+    indexDeveloper: `developer-wazo-io-${process.env.NODE_ENV}`,
+    indexPlatform: `wazo-platform-${process.env.NODE_ENV}`,
```

Comment:
We should not put code in a config file as the various CI system will not be aware of it and it will fail. I think it could be better to have this logic in `gatsby-node.js`.

Reply:
Moved to a new file `src/constants.js`, because indexes are used in `<Search />` components too.